src/datumaro/components/algorithms/rise.py

```diff
@@ -94,7 +94,8 @@ def apply(self, image, progressive=False):
             samples = min(self.max_samples, samples)
         batch_size = self.batch_size
 
-        result = next(iter(model.launch(_expand(image, 0))))
+        pred = next(iter(model.infer(_expand(image, 0))))
```

Comment:
Since the RISE code is already very outdated and would require significant modifications to get it right, I only made minimal changes following this refactoring.